modules/rosetta/src/main/scala/org/tessellation/rosetta/api/generated/dag/GenerateCodecs.scala

```diff
@@ -0,0 +1,27 @@
+package org.tessellation.rosetta.api.generated.dag
+
+import scala.reflect.io.Path
+import scala.tools.nsc.io.File
+
+/** This is a work-around for `sbt autogen` being broken (openAPI). @ryle-goehausen tried to get an autogen working, and this was the one
+  * that he found to be the least broken.
+  *
+  * TODO: Remove this when feasible.
+  */
+
+object GenerateCodecs {
```

Comment:
this is not auto generated but it is in `api.generated` package

Reply:
This is a utility to generate the stuff. I'll move it into a package adjacent to `org.tessellation.rosetta.api.generated`.

Reply:
You don't need to move it. Please see https://github.com/Constellation-Labs/tessellation/pull/543#discussion_r1073309026 and https://github.com/Constellation-Labs/tessellation/pull/543#discussion_r1073286301

Reply:
I will need assistance automating this task. Once that is done, this can be removed.

Reply:
After discussing this with @ryle-goehausen , I'll be removing this file. I will work with @tbekas on Friday to automate this if we can do so relatively quickly. If not, we'll just leave this out of the code base.

Reply:
The generation of the `Rosetta` classes has ben made a custom `sbt` task. This is no longer needed.